[2023-10-20 23:21:25,509] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2022-12-20T00:00:00+00:00 [queued]>
[2023-10-20 23:21:25,526] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2022-12-20T00:00:00+00:00 [queued]>
[2023-10-20 23:21:25,528] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-10-20 23:21:25,529] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-10-20 23:21:25,530] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-10-20 23:21:25,550] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): segunda_tarea> on 2022-12-20 00:00:00+00:00
[2023-10-20 23:21:25,554] {standard_task_runner.py:52} INFO - Started process 1521 to run task
[2023-10-20 23:21:25,558] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'segunda_tarea', 'scheduled__2022-12-20T00:00:00+00:00', '--job-id', '429', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmpcgzrms5j', '--error-file', '/tmp/tmprjoi0yf4']
[2023-10-20 23:21:25,560] {standard_task_runner.py:80} INFO - Job 429: Subtask segunda_tarea
[2023-10-20 23:21:25,650] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2022-12-20T00:00:00+00:00 [running]> on host dfdda228c9fc
[2023-10-20 23:21:25,746] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=segunda_tarea
AIRFLOW_CTX_EXECUTION_DATE=2022-12-20T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-20T00:00:00+00:00
[2023-10-20 23:21:25,748] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-10-20 23:21:25,749] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 2 y sere corrida luego de la Tarea 1']
[2023-10-20 23:21:25,758] {subprocess.py:85} INFO - Output:
[2023-10-20 23:21:25,760] {subprocess.py:92} INFO - hola, soy la tarea 2 y sere corrida luego de la Tarea 1
[2023-10-20 23:21:25,761] {subprocess.py:96} INFO - Command exited with return code 0
[2023-10-20 23:21:25,802] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=segunda_tarea, execution_date=20221220T000000, start_date=20231020T232125, end_date=20231020T232125
[2023-10-20 23:21:25,851] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-20 23:21:25,883] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
